src/renderer/fonts/caching_shaper.rs

```diff
@@ -171,6 +171,7 @@ impl CachingShaper {
             self.fudge_factor = font_width.round() / font_width;
             debug!("Fudge factor: {:.2}", self.fudge_factor);
             font_size = self.current_size();
+            self.font_info = None;
```

Comment:
I think this is still buggy, since `info()` is called below below in the debug statement, before the new font loader is created, `self.info` will therefore be cached with the old size again.

If that causes other problems though, then it's possible that the problem is something else than I inititially thought in https://github.com/neovide/neovide/issues/2214